ui/src/main/resources/org/pentaho/di/ui/spoon/messages/messages_ja_JP.properties

```diff
@@ -43,7 +43,7 @@ SpoonSlave.Column.Errors=\u30a8\u30e9\u30fc
 SpoonSlave.Column.Written=\u66f8\u304d\u8fbc\u307f
 Spoon.Dialog.ShowArguments.Message=\u5b9a\u7fa9\u3055\u308c\u305f\u5f15\u6570\u306e\u30ea\u30b9\u30c8\:
 Spoon.Dialog.WarningCreateNewSteps.Title=\u60c5\u5831
-Spoon.Menu.Edit.ShowArguments=\u5f15\u6570\u8868\u793a(&U)
+Spoon.Menu.Edit.ShowArguments=\u5f15\u6570\u8868\u793a(&U)...
```

Comment:
I don't have the faintest idea about this... are ellipsis used in Japanese?
I would say we should only change EN properties files.

Reply:
On the Japanese: yes and we've several examples in this same file.
On the "only EN": as I found several inconsistencies on the tag values for different languages and had them all available in my IDE at the time, I made the change; however I can remove them all.